compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/tree/BLangNodeVisitor.java

```diff
@@ -880,4 +880,8 @@ public void visit(BLangRecordLiteral.BLangRecordKeyValue recordKeyValue) {
     public void visit(BLangMarkdownReferenceDocumentation bLangMarkdownReferenceDocumentation) {
         throw new AssertionError();
     }
+
+    public void visit(BLangWaitForAllExpr.BLangWaitKeyValue waitKeyValue) {
```

Comment:
Do we really need to add this? @nadeeshaan , it seems like LS vistor is not using this. Shall we remove this?

Reply:
@hasithaa we use it at https://github.com/ballerina-platform/ballerina-lang/pull/19884/files#diff-e3e6d1e211484b619702615988df63b1R766